RebornCore/src/main/java/reborncore/common/crafting/ingredient/StackIngredient.java

```diff
@@ -101,7 +102,7 @@ public boolean test(ItemStack itemStack) {
 		}
 
 		if (nbt.isPresent()) {
-			if (!itemStack.hasNbt()) {
+			if (itemStack.getComponents() == ComponentMap.EMPTY) {
```

Comment:
I dont think this is correct, although im not 100% sure without checking.

ItemStacks have a number of default components that will always be present.

Reply:
Yeah, I haven't figured out yet if hasNbt() checks could be just removed